pkg/sdk/helper.go

```diff
@@ -219,6 +240,26 @@ func (a *SDKAPI) GetOperationMap(cfg *ackgenconfig.Config) *OperationMap {
 	return &opMap
 }
 
+// GetCustomListRef finds a ShapeRef for a supplied custom slice field
+func (a *SDKAPI) GetCustomListRef(memberShapeName string) (*awssdkmodel.ShapeRef, bool) {
```

Comment:
Thoughts on refactoring this to be a deeper method? ex:

```
func (a*SDKAPI) GetCustomShapeRef(shapeName string) (*awssdkmodel.ShapeRef, bool) {
    customShape, found := getCustomListRef(shapeName)
    if !found {
        customShape, found = getCustomMapRef(shapeName)
    }
    return customShape, found
}
```

Aligns with existing method names and easier to use for clients